packages/create-atomic/templates/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "{{project}}",
   "description": "{{titleCase project}} project",
   "version": "0.1.0",
+  "private": "true",
```

Comment:
if you were a developper using this, would you want the project to be private by default? I'm wondering... Is that what we do for other generated projects?

Reply:
I don't know if it's what we're doing today, but I do not think people would want to publish their project usually?

Reply:
Private by default is 👍 IMO